install/bruteforce.sh

```diff
@@ -0,0 +1,3 @@
+sudo apt-get install -y python-pip python-dev
+sudo apt-get install -y libatlas-dev libatlas3gf-base
+sudo apt-get install -y python-numpy
```

Comment:
Alright... can you check the timings for `np.dot` anyway, just to be sure? What version of numpy is that?